scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,183 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-- Despachiaire : !pos 108 -40 -83 26
+-- Rouva        : !pos -16 2 11 230
+-- ???          : !pos -412 0 -362 2
+-----------------------------------
+-- !addquest 4 75
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+-----------------------------------
+local carpentersLandingID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+local function AllParaDead()
+    for i = 0, 4 do
+        local mob = GetMobByID(carpentersLandingID.mob.PARA_OFFSET + i)
+        if not mob:isDead() then
+            return false
+        end
+    end
+
+    return true
+end
+
+quest.reward =
+{
+    item = xi.item.ELEGANT_RIBBON,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST) == QUEST_COMPLETED
```

Comment:
`player:hasCompletedQuest`

Reply:
Switched to hasCompletedQuest, had originally seen this in ASB and saw the same used in LSB (in a few quests but mainly in status == QUEST_COMPLETED, would it be helpful at all to make a pr switching to the hasCompletedquest pattern in these cases?)

Reply:
I'd say focus on converting first and replace with that pattern as you go.  There's a lot of overly complex conditions in some NPC scripts, so its probably best to handle one thing at a time